README.md

```diff
@@ -778,7 +780,7 @@ These people have contributed to improve this list for everyone (in no particula
 [36]:https://docs.google.com/spreadsheets/d/1wJwqv3rTNmORXz-XJsQaXK1dl8I91V4-eP_sfNVNzbA/edit "USB-C cable and Nexus accessories"
 [37]:https://de.wikipedia.org/wiki/Datenklo "CCC Datenklo - Akustik Koppler"
 [38]:https://cert.ccc.de/ "C3 CERT"
-[39]:https://events.ccc.de/congress/2016/wiki/Assembly:Coffeenerds "Coffeenerds at 33c3"
+[39]:https://events.ccc.de/congress/2023/hub/en/assembly/coffenerds/
```

Comment:
Should be `https://events.ccc.de/congress/2023/hub/fahrplan` without the `en/` which is picked by the browser.

Reply:
Indeed that's the better way, thanks for reviewing it.
I've added a new commit without the /en. (Same for the others)